paper/paper.md

```diff
@@ -14,29 +14,27 @@ authors:
     affiliation: "1, 2"
   - name: Martin Čech
     orcid: 0000-0002-9318-1781
-    affiliation: "1, 3"
+    affiliation: 1
   - name: Elliott James Price^[corresponding author]
     orcid: 0000-0001-5691-7000
     affiliation: 1
 
 affiliations:
- - name: RECETOX, Faculty of Science, Masaryk University, Kotlarska 2, Brno 60200, Czech Republic
+ - name: RECETOX, Faculty of Science, Masaryk University, Kotlářská 2, Brno 60200, Czech Republic
    index: 1
  - name: Institute of Computer Science, Masaryk University, Brno, Czech Republic
    index: 2
- - name: Institute of Organic Chemistry and Biochemistry of the CAS, Prague, Czech Republic
-   index: 3
 date: 16 December 2021
 bibliography: references.bib
 ---
 
 # Summary
 
-RIAssigner is a software package for the computation of gas chromatographic (GC) retention indices (RIs).
-The package uses matchms [@Huber2020] and pandas [@reback2020pandas] for data IO and supports `.msp` as well as tabular (`.csv` & `.tsv`) formats, among others.
+RIAssigner is a software package for the computation of gas chromatographic (GC) retention indices (RI).
+The package uses matchms [@Huber2020] and pandas [@reback2020pandas] for data I/O and supports the `.msp` as well as tabular (`.csv` & `.tsv`) formats, among others.
 It supports multiple keywords identifying the retention time (RT) and RI information and supports handling SI units for RT.
 The RI can be computed using non-isothermal Kováts retention-indexing (from temperature programming, using the definition of @VanDenDool:1963) or cubic spline interpolation [@Halang1978] based on a reference dataset containing RT & RI.
-The package is hosted via bioconda [@bioconda] and is accessible to users through the Galaxy ecosystem [@galaxy; @umsa].
+The MIT-licensed package is hosted via bioconda [@bioconda] and is also accessible to users as a Galaxy tool [@galaxy; @umsa].

```

Comment:
All JOSS packages must have an OSI approved license, so not sure whether mentioning it here is actually necessary.